ROBOT:
  hardware_interface:
     loop_hz: 50 
     joints:
       - j1
       - j2
       - j3
       - j4
